src/gsi/m_extOzone.F90

```diff
@@ -1021,7 +1039,7 @@ subroutine ozlev_ncread_(dfile,dtype,ozout,nmrecs,ndata,nodata, gstime,twind)
            if (ozone(ilev, iprof) < -900.0_r_kind) cycle            ! undefined
            if (err(ilev, iprof) < -900.0_r_kind) cycle              ! undefined
            if (iuse_oz(ipos(ilev)) < 0) then
-              usage = 100._r_kind
+              usage = 10000._r_kind
```

Comment:
what is the reason for changing the usage from 100 to 10000? This section is in the subroutine ozlev_ncread_. There is a similar subroutine ozlev_bufrread_ where the usage is still set to 100. I am unsure the impact of changing the usage from 100 to 10000, but it is important to ensure consistency for the same variables in both subroutines.